ntbs-service/Pages/Notifications/LegacyView.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@page "/Notifications/LegacyView/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.Notifications.LegacyView
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
```

Comment:
Could we use a layout like _NotificationLayout in this file instead of putting the banner in manually?

Reply:
_NotificationLayout is tightly coupled with NotificationModelBase and LegacyView does not extend it, therefore I had to write banner here instead of reusing layout

Reply:
Ah nice ofc! In that case happy to leave as it is unless there are going to be any more pages like this one that would need a similar layout!